database/migrations/2021_09_20_152502_add_force_change_password_to_users_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddForceChangePasswordToUsersTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('users', function (Blueprint $table) {
+            $table->boolean('force_change_password')->default(1);
```

Comment:
Correct me if I am wrong, but once this migration is executed, the old users should stay with their current password and should no be forced to change it.
```suggestion
            $table->boolean('force_change_password')->default(0);
```

Reply:
It is not specified in the ticket the initial status, but this makes sense. I will change it and do a test